include/certifier_framework.h

```diff
@@ -42,35 +43,71 @@ namespace certifier {
       key_message policy_key_;
       string encryption_algorithm_;
 
+      // -------------------------------------------------------------------
+      // Different types of artifacts that are tracked in the Policy Store
+      // -------------------------------------------------------------------
+      //
+      trusted_service_message** ts_;
       int max_num_ts_;
       int num_ts_;
-      trusted_service_message** ts_;
```

Comment:
I relocated this to the L50 above, as it made better sense to list the data structure first for which the max/num-related fields appear afterward.